### ISSUE TITLE
example errors

### ISSUE THREAD

**Issue by ghost** [foomo/simplecert#7]
Hi,

Hope you are all well !

Examples are failing as the following error is triggered:

```bash
# command-line-arguments
./simplecert.go:34:38: not enough arguments in call to simplecert.Init
	have (*simplecert.Config)
	want (*simplecert.Config, func())
```

Any way to update to them ?

Cheers,
X

**Comment by ernsheong** [comment]
You can just do this for now: 

```
certReloader, err := simplecert.Init(cfg, nil)
```

**Comment by ghost** [comment]
yeah, that's what I did but the README needs to be fixed also

**Comment by dreadl0ck** [comment]
Hi, thanks for the heads up, fixed with commit e438243.
